.circleci/config.yml

```diff
@@ -103,6 +103,23 @@ commands:
       - run:
           name: Running network tests
           command: npm run test:docker
+  extension-server-start:
+    steps:
+      - run:
+          name: "Extension: server start"
+          background: true
+          command: |
+            echo 'export APPLICATION_SID_EXTENSION="$APPLICATION_SID_EXTENSION"' >> "$BASH_ENV"
+            echo 'export ACCOUNT_SID="$ACCOUNT_SID"' >> "$BASH_ENV"
+            echo 'export API_KEY_SID="$API_KEY_SID"' >> "$BASH_ENV"
+            echo 'export API_KEY_SECRET="$API_KEY_SECRET"' >> "$BASH_ENV"
+            source "$BASH_ENV"
```

Comment:
These lines should be unnecessary. We're basically reading env vars, and then setting env vars. So we're redeclaring the same env vars with the same values and these lines are not really doing anything.

Reply:
Good catch, export not necessary 👍